src/preamble.js

```diff
@@ -1427,10 +1445,11 @@ var __ATMAIN__    = []; // functions called when main() is to be run
 var __ATEXIT__    = []; // functions called during shutdown
 var __ATPOSTRUN__ = []; // functions called after the runtime has exited
 
-var runtimeInitialized = false;
+var runtimeInitialized = ENVIRONMENT_IS_PTHREAD; // The runtime is hosted in the main thread, and bits shared to pthreads via SharedArrayBuffer. No need to init again in pthread.
```

Comment:
got it, but how about setting to false, then if it's a pthread, set it to true a few lines down? that seems clearer, as the semantic meaning of `runtimeInitialized` is not `is a pthread`